### DISCUSSION TITLE
Auto Import or Import/Export XML or Remember Directory

### DISCUSSION THREAD

**Discussion by Querzion** [Heroic-Games-Launcher/HeroicGamesLauncher#793] [Ideas]
I have installed almost all my games, and it takes a lot of time to switch from ready to all or not ready after EVERY imported game. Are there more ppl out there that would had liked an add in features?

Auto Import from chosen Install Folder or
Import/Export XML or JSON or something that saves the games directory for Auto Import.
It would also be nice if the last directory imported from could be in memory, since I have a long path for my installs, but I have added shortcuts in order to get into the folder faster, still a lot of extra work.

**Comment by flavioislima** [comment]
And why don't you use the Sync with EGL feature to import all games at once?

**Reply by Querzion** [reply]
It has a sync function?
